src/app.service.ts

```diff
@@ -111,7 +112,28 @@ export class AppService {
       .addBearerAuth()
       .build();
 
-    const document = SwaggerModule.createDocument(app, config, {});
+    const document = SwaggerModule.createDocument(app, config, {
+      operationIdFactory: (controllerKey: string, methodKey: string) => {
+        const replacedControllerName = controllerKey.replace(/Controller$/, '');
+
+        const singularControllerName = inflection.singularize(
+          replacedControllerName,
+        );
+
+        const singularMethodName = inflection.singularize(methodKey);
+
+        const methodName = singularMethodName.replace(
+          singularControllerName,
+          '',
+        );
+
+        const controllerName =
+          singularControllerName.charAt(0).toLowerCase() +
+          singularControllerName.slice(1);
```

Comment:
```suggestion
 const controllerName = singularControllerName.replace(
          /^(.)/,
          (matchStr) => matchStr.toLowerCase(),
        );
```
이런식으로도 가능한데 더 좋아보이는거로